src/aap_eda/api/serializers/activation.py

```diff
@@ -143,6 +143,8 @@ def to_representation(self, activation):
 class ActivationCreateSerializer(serializers.ModelSerializer):
     """Serializer for creating the Activation."""
 
+    organization_id = serializers.IntegerField(required=False, allow_null=True)
```

Comment:
Why do we need this?

Reply:
You are looking at the `ActivationCreateSerializer`. This is used in the view create code to create `serializer`

https://github.com/ansible/eda-server/blob/41e181dee75467e0d75810f998871d0723a19508/src/aap_eda/api/views/activation.py#L83-L85

And then from `serializer`, a `response` is created, which is really the new activation object, in `response = serializer.create(serializer.validated_data)`.

If `organization_id` is not added as a serializer field, then it cannot be in `validated_data`. I know there is some confusion about this because of the use of `ModelSerializer`, which automatically builds fields from the Django model definition. But that would be `organization` that gets automatically built, which is not listed in fields of this serializer. So for the created object to have the organization specified in the request data, this is needed. Similar patterns already exist in other serializers.

Reply:
Hmm.. we have `organization_id` listed in Meta.fields. I thought that would be enough to have the organization id surfaced since we are using ModelSerializer.